tools/coverity.sh

```diff
@@ -135,8 +140,12 @@ _upload()
 	  --form version=$SHA \
 	  --form description="Travis CI build" \
 	  $UPLOAD_URL)
+	printf "\033[33;1mThe response is\033[0m\n%s\n" "$response"
 	status_code=$(echo "$response" | sed -n '$p')
-	if [ "$status_code" != "201" ]; then
+	# Coverity Scan used to respond with 201 on successfully receieving analysis results.
+	# Now for some reason it sends 200 and may change back in the foreseeable future.
+	# See https://github.com/pmem/pmdk/commit/7b103fd2dd54b2e5974f71fb65c81ab3713c12c5
+	if [ "$status_code" != "200" ]; then
```

Comment:
Suggestion: Perhaps accept either 200 or 201 as success?

Reply:
Thank you. I was thinking about this because I didn't want the script to break suddenly. On the other hand, Coverity Scan sends `200` even when the token isn't provided, so I decided it'd be better to accept 200 for now and remove it as soon as the API is fixed. It seems that the transition will be easier to catch if the script starts fail on receiving `200`.